R/dsl.R

```diff
@@ -455,3 +455,53 @@ menu_header <- function(..., is_item = TRUE) {
 menu_divider <- function(...) {
   div(class = "divider", ...)
 }
+
+#' Helper function to render list element
+#'
+#' @param header String with header
+#' @param description String with description
+#' @param icon Icon string
+list_element <- function(data, is_description, icon, row){
+  div(class = "item",  if(icon == "") "" else uiicon(icon),
```

Comment:
In line 500 you use `ifelse`. Better be consistent with one liners.

Reply:
The thing is it's not possible to ude ifelse here, since ifelse for single TRUE condition `icon != ""` will return the single element from uiicon(icon) which is `i`  from `<i class="alarm icon"></i>`. I don't heve this problem with if, so that's why I decided to use it here.

Reply:
👍